libraries/RC_Channel/RC_Channels_VarInfo.h

```diff
@@ -87,7 +90,7 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @Description: RC input options
     // @User: Advanced
     // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse
-    AP_GROUPINFO("_OPTIONS", 33, RC_CHANNELS_SUBCLASS, _options, 0),
+    AP_GROUPINFO("_OPTIONS", 33, RC_CHANNELS_SUBCLASS, _options, (const float)RC_Channels::Option::ARMING_CHECK_THROTTLE),
```

Comment:
This would cause a change for the default for tradheli.  I realize we are talking about a safety issue for copter but this is not a safety issue for tradheli.  @rmackay9  how can this be resolved to not change current and future setups for tradheli?

Reply:
good catch, we should just check for heli frame with #ifdef in AP_Arming.cpp for now